### ISSUE TITLE
Increase background colour contrast of currently selected item

### ISSUE THREAD

**Issue by alexanderadam** [amandamcg/joplin-theme#5]
What do you think about increasing the contrast, so that it is better visible what's currently selected?
Maybe also for
- the scrollbar
- selected text

**Comment by AishwaryaRavi07** [comment]
Hi, I'm interested in working on this, could you please tell me what exactly do you need this feature?

**Comment by alexanderadam** [comment]
I don't think this theme is maintained any more hence I switched to [another one](https://github.com/Sinacs/Joplin.Plugin.ReMoods.Theme#remoods-theme).
